mgradm/cmd/distro/cp.go

```diff
@@ -29,10 +28,12 @@ func umountAndRemove(mountpoint string) {
 	}
 
 	if err := utils.RunCmd("/usr/bin/sudo", umountCmd...); err != nil {
-		log.Fatal().Err(err).Msgf(L("Unable to unmount ISO image, leaving %s intact"), mountpoint)
+		log.Error().Err(err).Msgf(L("unable to unmount iso file, leaving %s intact"), mountpoint)
```

Comment:
Why are you changing the message here? It seems like a rebase issue

Reply:
I was primarily getting rid of `Fatal` call. Reworded back, keeping error.

Reply:
well, the message is not the same :wink: No need to change the message itself.
```suggestion
		log.Error().Err(err).Msgf(L("Unable to unmount ISO image, leaving %s intact"), mountpoint)
```